tests/tests_suite_test.go

```diff
@@ -45,9 +48,27 @@ var _ = BeforeSuite(func() {
 	Expect(err).ToNot(HaveOccurred())
 	clientset, err = kubernetes.NewForConfig(config)
 	Expect(err).ToNot(HaveOccurred())
+	createTestNamespace()
```

Comment:
We don't need the namespace so far.

Reply:
I will like to add this so we can be sure everything we run the tests we don't leave anything. (I know in the std-ci we get a fresh environment every time but not when we run make cluster-sync and make functests)

Reply:
My point is the namespace is not used for anywhere in the test.
It's created/deleted, but not used. The marker is created in a different namespace outside the test. 

It could be useful if we add more tests ...

Reply:
Oh you are right sorry I was thinking about adding a test that start a pod but then drop the idea as it is not relevant to the project(this project only add the capacity to the node) the resource in the pod should be configured by the user on the scheduling is part of k8s.

I will drop this thanks!